gprofiler/profilers/factory.py

```diff
@@ -42,7 +42,7 @@ def get_profilers(
         except Exception:
             if profiler_config.profiler_class is SystemProfiler:
                 raise SystemProfilerInitFailure("Could not create the system profiler")
-            logger.exception(f"Couldn't create the {profiler_name} profiler, continuing without it")
+            logger.critical(f"Couldn't create the {profiler_name} profiler, continuing without it", exc_info=True)
```

Comment:
This is the important one, why don't we exit with error on this, suggesting to run with `--no-{profiler_name}` if user wants to ignore?

Reply:
Agree, it should exit here probably. Will change

Reply:
Done